api/v1alpha1/fenceagentsremediation_webhook.go

```diff
@@ -52,13 +54,19 @@ var _ webhook.Validator = &FenceAgentsRemediation{}
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (far *FenceAgentsRemediation) ValidateCreate() (admission.Warnings, error) {
 	webhookFARLog.Info("validate create", "name", far.Name)
-	return validateAgentName(far.Spec.Agent)
+	if _, err := validateAgentName(far.Spec.Agent); err != nil {
```

Comment:
would be nice to run all validations and return an aggregated error, see e.g. https://github.com/medik8s/node-healthcheck-operator/blob/a295befac6e76e4dce4d92fdd2093b6ebf39239a/api/v1alpha1/nodehealthcheck_webhook.go#L93

Reply:
Not yet fixed it, but i will change it to the aggregated error in NHC.